roles/storage-demo/templates/storage-demo.yml

```diff
@@ -50,7 +50,7 @@ spec:
         args: ["demo"]
         env:
         - name: MON_IP
-          value: "{{ storage_demo_node_hostname }}"
+          value: "{{ storage_demo_node_ip }}"
```

Comment:
Don't forget to fix it everywhere in the manifest

Reply:
Also, did you try:
```
      - name: kraken                                                            
        args: ["demo"]                                                          
        env:                                                                    
        - name: MON_IP                                                          
          valueFrom:                                                            
            fieldRef:                                                           
              fieldPath: status.hostIP                                          
```

Reply:
yeah, this format is same as ceph-cinder-demo, but it's seems to have some validation problem in cli or apiserver, I'll try to figure it out.